tests/class-two-factor-core.php

```diff
@@ -37,41 +42,62 @@ public static function tearDownAfterClass() {
 		parent::tearDownAfterClass();
 	}
 
+	/**
+	 * Print error messages and return true if error is a notice
+	 *
+	 * @param integer $errno error number.
+	 * @param string  $errstr error message text.
+	 *
+	 * @return boolean
+	 */
 	public static function error_handler( $errno, $errstr ) {
-		if ( E_USER_NOTICE != $errno ) {
-			echo 'Received a non-notice error: ' . $errstr;
+		if ( E_USER_NOTICE !== $errno ) {
+			echo 'Received a non-notice error: ' . esc_html( $errstr );
 
 			return false;
 		}
 
 		return true;
 	}
 
+	/**
+	 * Get a dummy user object.
+	 *
+	 * @param array $meta_key authentication method.
+	 *
+	 * @return WP_User
+	 */
 	public function get_dummy_user( $meta_key = array( 'Two_Factor_Dummy' => 'Two_Factor_Dummy' ) ) {
-		$user = new WP_User( $this->factory->user->create() );
+		$user              = new WP_User( $this->factory->user->create() );
 		$this->old_user_id = get_current_user_id();
 		wp_set_current_user( $user->ID );
 
-		$key = '_nonce_user_two_factor_options';
-		$_POST[$key] = wp_create_nonce( 'user_two_factor_options' );
-		$_REQUEST[$key] = $_POST[$key];
+		$key              = '_nonce_user_two_factor_options';
+		$nonce            = wp_create_nonce( 'user_two_factor_options' );
+		$_POST[ $key ]    = $nonce;
+		$_REQUEST[ $key ] = $nonce;
 
-		$_POST[Two_Factor_Core::ENABLED_PROVIDERS_USER_META_KEY] = $meta_key;
+		$_POST[ Two_Factor_Core::ENABLED_PROVIDERS_USER_META_KEY ] = $meta_key;
 
 		Two_Factor_Core::user_two_factor_options_update( $user->ID );
 
 		return $user;
 	}
 
+	/**
+	 * Clean up the dummy user object data.
+	 */
 	public function clean_dummy_user() {
-		unset( $_POST[Two_Factor_Core::ENABLED_PROVIDERS_USER_META_KEY] );
+		unset( $_POST[ Two_Factor_Core::ENABLED_PROVIDERS_USER_META_KEY ] ); // phpcs:ignore WordPress.Security.NonceVerification.Missing
```

Comment:
Could we disable `WordPress.Security.NonceVerification` for all tests by default?

Reply:
Done.